### PR TITLE
virtio_serial:boot guest with multiple serials

### PR DESCRIPTION
**Pull request by nanliu-r**
Boot a guest with one virtio-serial-pci with three serial ports

ID:1588426 

Signed-off-by: nanliu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
Test PASS:

 (1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.unix_socket.boot_with_multiple_serials.1_N: PASS (165.12 s)

**Comment by nanliu-r** [comment]
@vivianQizhu  Please help to review this, thanks a lot!

File: qemu/tests/boot_1_N.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    os_type = params["os_type"]
+    vm = env.get_vm(params["main_vm"])
+    driver_name = params["driver_name"]
+    session = vm.wait_for_login()
+    transfer_flag = False
+    if os_type == "windows":
+        session = utils_test.qemu.windrv_check_running_verifier(
+            session, vm, test, driver_name)
+    for port in params.objects("serials"):
```
Comment (vivianQizhu) [inline qemu/tests/boot_1_N.py]: What about `params.objects("serials")[2:]`? Not need transfer_flag then.

Comment (nanliu-r) [reply qemu/tests/boot_1_N.py]: > What about `params.objects("serials")[2:]`? Not need transfer_flag then.

Thanks and update .

File: qemu/tests/cfg/virtio_serial_file_transfer.cfg [hunk 2]
```diff
@@ -28,6 +28,15 @@
                     file_sender = host
                 - from_guest_to_host:
                     file_sender = guest
+        - boot_with_multiple_serials:
+            variants:
+                - 1_N:
+                    only unix_socket
+                    type = boot_1_N
```
Comment (vivianQizhu) [inline qemu/tests/cfg/virtio_serial_file_transfer.cfg]: Script name at least include feature or device name, thanks.

Comment (nanliu-r) [reply qemu/tests/cfg/virtio_serial_file_transfer.cfg]: > Script name at least include feature or device name, thanks.

Thanks and update.

**Comment by vivianQizhu** [comment]
Ack

**Pull request merged**